src/components/widgets/camera/services/WebrtcCamerastreamerCamera.vue

```diff
@@ -83,6 +84,14 @@ export default class WebrtcCamerastreamerCamera extends Mixins(CameraMixin) {
 
       const pc = this.pc = new RTCPeerConnection(config)
 
+      pc.ondatachannel = (event: RTCDataChannelEvent) => {
+        const dc = event.channel
+
+        dc.onmessage = () => {
```

Comment:
Can you check that this dc is named `keepalive` and send only on this one?

Reply:
@pedrolamas I will extend my changes to include `dc.label == 'keepalive'` check. If at somepoint there is a proper data channel for control messages it would not interfere with the clients.